src/components/legalLinks/legalLinkModal.styles.scss

```
@@ -0,0 +1,37 @@
+.legalLinkModal__content {
+	color: #1e1e1e;
+	font-size: 16px;
+	line-height: 1.4;
+	white-space: normal;
+	word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)